HW_1/Task_4.py

```diff
@@ -0,0 +1,28 @@
+# Задача 8:
+# Требуется определить,
+# 1. можно ли от шоколадки размером n × m долек отломить k долек,
+# 2. если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+number1 = int(input("Введите длину шоколадки n: "))
+number2 = int(input("Введите ширину шоколадки m: "))
+
+number1_2 = int(number1*number2)
```

Comment:
Не нужно переводить в целочисленный тип, если вы перемножаете 2 целочисленных типа.
```suggestion
number1_2 = number1 * number2
```